modules/nf-core/seurat/multiseq/environment.yml

```
@@ -0,0 +1,9 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+name: "seurat_multiseq"
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - "bioconda::r-seurat=3.0.2"
```

```suggestion
channels:
  - conda-forge
  - bioconda
dependencies:
  - "bioconda::r-seurat=3.0.2"
```
Update to newest style